src/NuGet.Core/NuGet.Configuration/Settings/Items/TrustedSignerItem.cs

```diff
@@ -0,0 +1,79 @@
+// Copyright(c) .NET Foundation.All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Xml.Linq;
+
+namespace NuGet.Configuration
+{
+    public abstract class TrustedSignerItem : SettingItem
+    {
+        public string Name => Attributes[ConfigurationConstants.NameAttribute];
+
+        protected override bool CanHaveChildren => true;
+
+        private IList<CertificateItem> _certificates { get; }
+
+        public IReadOnlyList<CertificateItem> Certificates => _certificates.ToList().AsReadOnly();
+
+        protected TrustedSignerItem(string name, IEnumerable<CertificateItem> certificates)
+            : base()
+        {
+            if (string.IsNullOrEmpty(name))
+            {
+                throw new ArgumentNullException(nameof(name));
+            }
+
+            AddAttribute(ConfigurationConstants.NameAttribute, name);
+
+            _certificates = new List<CertificateItem>();
```

Comment:
To avoid null checking everywhere else I want to make sure there is always a list (even if it's empty). This is the same pattern followed in `UnknownItem`, `SettingGroup` and `SettingElement`.